docs/api/sql/Raster-visualizer.md

```diff
@@ -68,6 +68,14 @@ RS_AsMatrix expects a raster, and optionally a band (default: 1) and postDecimal
 !!!Note
     If the provided raster has integral values, postDecimalPrecision (if any) is simply ignored and integers are printed in the resultant string
 
+!!!note
+    If you are using `show()` to display the output, it will show special characters as escape sequences. To get the expected behavior use the following code:
+
+    ```sql
+    print(df.selectExpr("RS_AsMatrix(rast)").sample(0.5).collect().mkString(""))
```

Comment:
This is not a SQL example. This should be a language-specific example. You need to provide examples for (Scala/Java) and (Python). Please also say `sample()` here is to reduce data to be collected. Other functions such as filter() will work as well.